src/uhs/client/client.hpp

```diff
@@ -237,6 +251,8 @@ namespace cbdc {
         /// \brief Sends the given minting transaction to a service that will
         ///        accept and process it.
         ///
+        /// TODO:  Remove. No longer needed
```

Comment:
@HalosGhost  I primarily left it in the event someone objected to the changes and/or wanted the functionality for other reasons (testing?).

Reply:
Nothing occurs to me on why it would still be needed with the adopting of this (pretty simple even) minting mechanism, but perhaps @metalicjames will think otherwise (I'm open to hearing alternative opinions). :)

Reply:
If we're not using it, let's remove it.